radio/src/opentx.h

```diff
@@ -847,6 +847,11 @@ void checkAll();
 
 #if !defined(SIMU)
 void getADC();
+  #if defined(CPUARM)
+    #define getADC_ifMixerNotRunning()    do { if (s_pulses_paused) getADC(); } while(0)
```

Comment:
I like this solution, yes
Instead I don't like this function name which mixes all existing styles around the world :)
May I propose GET_ADC_IF_MIXER_NOT_RUNNING()?